playground/TestShop/TestShop.ServiceDefaults/Extensions.cs

```diff
@@ -52,7 +55,12 @@ public static TBuilder ConfigureOpenTelemetry<TBuilder>(this TBuilder builder) w
             })
             .WithTracing(tracing =>
             {
-                tracing.AddAspNetCoreInstrumentation()
+                tracing
+                    .AddAspNetCoreInstrumentation(tracing =>
```

Comment:
Do you think it's worth adding a test to playground that validates that health checks are being filtered and not sent through OTLP or is that overkill?

Reply:
I think that's overkill. We're just using standard APIs from OTel for ASP.NET Core. If we wanted a test it would be on templates I'd think.

Reply:
#8645